### DISCUSSION TITLE
这个和mix space是什么关系呢

### DISCUSSION THREAD

**Discussion by xingpingcn** [Innei/Shiro#443] [General]
现在的话我用的是hexo，是通过本地构建然后部署到vercel的。

看了一下本主题的部署文档，感觉略麻烦，且在在docker的情况下还需要mix space吗

**Comment by TNXG** [comment]
基于Mix-Space，Mix-Space为Shiro的后端，Shiro则为Mix-Space的前端。~~应该是~~

**Reply by yizhimao213** [reply]
前后端分离，hexo是静态

**Reply by yizhimao213** [reply]
回错帖子了，算了

**Reply by xingpingcn** [reply]
恩。我晚上看了一下，确实是这样的。

我很喜欢这个主题，很漂亮，我想把现在的博客迁移过去，发现这个主题需要后端。本来我想搭一个试试的，但是我看docker compose里面需要安装两个数据库，我觉得有点重了。

且我再看主题的特性里面有实时统计在线人数，和自带访客统计等等，还有其他基于ws的特性，我觉得这些特性对于我来说可有可无。

于是我想利用nextjs的ssg，像使用hexo那样生成静态博客，但是我发现主题似乎没有提供纯静态的版本，而且我对nextjs也是不懂的，不知道维护者有没有出静态版本的打算

**Reply by Innei** [reply]
暂时没有打算
